docs/getting_started.md

```diff
@@ -24,17 +24,17 @@ This is a guide on how to get started with CAPV (Cluster API Provider vSphere).
 
 #### clusterctl
 
-Please download the latest `clusterctl` from the Cluster API (CAPI), GitHub [releases page](https://github.com/kubernetes-sigs/cluster-api/releases).
+Download the latest `clusterctl` from the Cluster API (CAPI), GitHub [releases page](https://github.com/kubernetes-sigs/cluster-api/releases).
 
-#### Docker
+##### Docker
```

Comment:
Curious why the extra # here? The header size should match `clusterctl` since they are all prerequisites.